css-forms-1/Overview.bs

```diff
@@ -677,14 +677,18 @@ input:is([type=checkbox]:not([switch]), [type=radio]) {
     display: inline-flex;
     align-items: center;
     justify-content: center;
+    content: '';
+}
+
+input[type=radio] {
+		border-radius: 100%;
 }
 
-input[type=checkbox]:not([switch])::checkmark {
+input[type=checkbox]:not([switch]):checked::checkmark {
     content: '\2713' / '';
 }
 
-input[type=radio]::checkmark {
-    content: '';
+input[type=radio]:checked::checkmark {
     background-color: currentColor;
     display: inline-block;
     border-radius: 100%;
```

Comment:
```suggestion
    border-radius: inherit;
```

In the unlikely event where someone changes the border-radius of the radio button, this might be useful.